setup.cfg

```diff
@@ -21,3 +21,10 @@ exclude =
     */south_migrations/,
     */migrations/,
     */testsettings.py,
+
+
+[tool:pytest]
```

Comment:
Ik heb er vaak deze nog bijstaan:

    addopts = --flakes --cov --cache-clear lizard_auth_server/

De `--opties` zijn ter info, het gaat me om het laatste ding. Dan zoekt pytest alleen in die directory en hoef je dus niet meer de `norecursedirs` up to date te houden.

Reply:
Ik heb flakes er even uit gelaten, heb mijn setup nu met flake8 gedaan en die checkt weer net anders.